src/Control/Distributed/Spark/RDD.hs

```diff
@@ -92,6 +122,27 @@ aggregate seqOp combOp zero rdd = do
   res :: JObject <- call rdd "aggregate" [coerce jzero, coerce jseqOp, coerce jcombOp]
   reify (unsafeCast res)
 
+treeAggregate :: ( Reflect (Closure (b -> a -> b)) ty1
```

Comment:
Let's keep the indentation consistent with what the rest of the module already uses.